internal/v1/handler_blueprints.go

```diff
@@ -144,6 +147,88 @@ func (u *User) MergeForUpdate(userData []User) error {
 	return nil
 }
 
+// Helper function to build service snapshots if compliance policy exists
+func (h *Handlers) buildServiceSnapshots(ctx echo.Context, customizations *Customizations, distribution Distributions) (*db.ServiceSnapshots, error) {
```

Comment:
Could you split out any api changes to a separate commit? It makes it a bit hard to review.